external/kinesis-asl/src/main/scala/org/apache/spark/examples/streaming/KinesisWordCountASL.scala

```diff
@@ -78,7 +78,7 @@ object KinesisWordCountASL extends Logging {
     if (args.length != 3) {
       System.err.println(
         """
-          |Usage: KinesisWordCountASL <app-name> <stream-name> <endpoint-url> <region-name>
```

Comment:
I'm probably mixed up but you remove the region name arg here, but then add it to the example command?

Reply:
`KinesisWordCountASL.scala` doesn't require the fourth argument (region) because it's derived from the third argument (endpoint URL) using [the `getRegionNameByEndpoint` function](https://github.com/apache/spark/blob/master/external/kinesis-asl/src/main/scala/org/apache/spark/examples/streaming/KinesisExampleUtils.scala). On the other hand, `kinesis_wordcount_asl.py` requires the fourth argument because it doesn't have that function. So I removed that parameter from the Scala example and added it to the Python example.